cves/kernel/CVE-2018-13406.yml

```diff
@@ -335,9 +363,9 @@ stacktrace:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  any_stacktraces:
-  stacktrace_with_fix:
-  note:
+  any_stacktraces: false
+  stacktrace_with_fix: false
+  note: None, because the Linux Kernel is written in C.
```

Comment:
Same comment as above. Although I'm surprised you didn't encounter any stacktraces on this one.